joomla/joomla-cms
# [4.0] Workflows checked out / checkin

# Summary of Changes
Checked_out and Checkin now works in com_workflow in the same way as in all other components


### Testing Instructions
Open a workflow or stage or transition, close the browser window. Login again and see the locked icon in list views.
Klick on the locked icon to chckin the item. If you have permission for checkin, the item will be unlocked.
